src/components/UI/Aside/Aside.css

```diff
@@ -36,14 +36,18 @@
 }
 
 .nav--mobile .nav__items--mobile {
-	margin-left: 0;
-	display: flex;
-	flex-flow: row nowrap;
-	list-style-type: none;
-	justify-content: space-evenly;
-	align-items: center;
-	width: 100%;
-	color: var(--COLOR-TXT);
+    margin-left: 0;
+    display: flex;
+    flex-flow: row nowrap;
+    list-style-type: none;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 100%;
+    position: fixed;
+    bottom: 4.6rem;
+    padding: 1.2rem 0;
+    background: var(--COLOR-BG);
```

Comment:
give fallback values in when using variables

Reply:
will look into it.